src/utils/index.js

```diff
@@ -2,13 +2,15 @@ import * as React from 'react'
 import { BigNumber } from 'bignumber.js'
 import dayjs from 'dayjs'
 import utc from 'dayjs/plugin/utc'
+import relativeTime from 'dayjs/plugin/relativeTime'
 import { Text } from 'rebass'
 import Numeral from 'numeral'
 import { timeframeOptions } from '~/constants'
 export * from './blockExplorers'
 
 BigNumber.set({ EXPONENTIAL_AT: 50 })
 dayjs.extend(utc)
+dayjs.extend(relativeTime)
```

Comment:
`relativeTime` is a small plugin, less than 100 LOC, so I think we're fine adding it

https://github.com/iamkun/dayjs/blob/dev/src/plugin/relativeTime/index.js